oca_dependencies.txt

```diff
@@ -0,0 +1,7 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+partner-contact
+server-tools
+stock-logistics-workflow
+webkit-tools
+web
```

Comment:
We only need `server-tools` here (the module depends on `configuration_helper` from OCA/server-tools and `base_delivery_carrier_label` depending on `delivery` only which is a standard one).

Reply:
Please attend this comment for not making slower this CI